test/Graphics/Texture.test.cpp

```diff
@@ -24,4 +25,219 @@ TEST_CASE("[Graphics] sf::Texture" * doctest::skip(skipDisplayTests))
         CHECK(!texture.isRepeated());
         CHECK(texture.getNativeHandle() == 0);
     }
+
+    SUBCASE("create()")
+    {
+        sf::Texture texture;
+
+        CHECK(texture.create(sf::Vector2u({123, 456})));
+
+        CHECK(texture.getSize() == sf::Vector2u({123, 456}));
+    }
+
+    SUBCASE("Copy semantics")
+    {
+        constexpr std::uint8_t red[] = {0xFF, 0x00, 0x00, 0xFF, 0xFF, 0x00, 0x00, 0xFF};
+
+        sf::Texture texture;
+
+        REQUIRE(texture.create(sf::Vector2u({1, 2})));
+
+        texture.update(red);
+
+        SUBCASE("Construction")
+        {
+            const sf::Texture textureCopy(texture);
+
+            REQUIRE(textureCopy.getSize() == sf::Vector2u({1, 2}));
+            CHECK(textureCopy.copyToImage().getPixel(sf::Vector2u({0, 1})) == sf::Color::Red);
+        }
+
+        SUBCASE("Assignment")
+        {
+            sf::Texture textureCopy;
+            textureCopy = texture;
+
+            REQUIRE(textureCopy.getSize() == sf::Vector2u({1, 2}));
+            CHECK(textureCopy.copyToImage().getPixel(sf::Vector2u({0, 1})) == sf::Color::Red);
+        }
+    }
+
+    SUBCASE("update()")
+    {
+        constexpr std::uint8_t yellow[] = {0xFF, 0xFF, 0x00, 0xFF};
+        constexpr std::uint8_t cyan[]   = {0x00, 0xFF, 0xFF, 0xFF};
+
+        sf::Texture texture;
+
+        SUBCASE("Pixels")
+        {
+            REQUIRE(texture.create(sf::Vector2u({1, 1})));
+
+            texture.update(yellow);
+
+            CHECK(texture.copyToImage().getPixel(sf::Vector2u({0, 0})) == sf::Color::Yellow);
+        }
+
+        SUBCASE("Pixels, size and destination")
+        {
+            REQUIRE(texture.create(sf::Vector2u({2, 1})));
+
+            texture.update(yellow, sf::Vector2u({1, 1}), sf::Vector2u({0, 0}));
+            texture.update(cyan, sf::Vector2u({1, 1}), sf::Vector2u({1, 0}));
```

Comment:
```suggestion
            texture.update(cyan, sf::Vector2u(1, 1), sf::Vector2u(1, 0));
```
You can omit also these extra braces you put inside the `sf::Vector2` constructor calls.